amazon-link-engine/trunk/amazon-link-engine.php

```diff
@@ -147,7 +149,7 @@ function genius_ale() {
 }
 
 if (!is_admin()) {
-  add_action('wp_head', 'genius_ale');
+  add_action('wp_head', 'genius_ale', 9999);
```

Comment:
Yeah, the snippet was loading below my jquery so not having it as 9999 caused an issue, I wouldn't say this snippet is needed but might help some... the other bit of JS is essential in the pull though.